lahja/endpoint.py

```diff
@@ -198,12 +198,8 @@ def name(self) -> str:
     @property
     def has_snappy_support(self) -> bool:
         if self._has_snappy_support is None:
-            try:
-                import snappy  # noqa: F401
-                self._has_snappy_support = True
-            except ModuleNotFoundError:
-                self._has_snappy_support = False
-
+            from lahja._snappy import is_snappy_available
+            self._has_snappy_support = is_snappy_available
```

Comment:
Then this would become `self._has_snappy_support = has_snappy_support()`. There's probably something clever that I'm missing but from the top of my head I don't see the benefit of that construct.

Reply:
I think I'm :+1: with deciding at import time since this isn't something that should change over the lifecycle of the python process

Reply:
Ah, so now I understand the reason for that other approach. But yeah, I think assuming snappy support is available at the time when the process starts is a reasonable assumption to make.